Replace bluetooth Readme with corrected version from devices repository

atouchet opened the pull request:
This Readme is for a different crate located in https://github.com/servo/servo/tree/main/third_party/blurmac. Was adding it to the bluetooth directory in #30974 a mistake?

atouchet committed:
Remove Readme from components/bluetooth

Signed-off-by: Alex Touchet <26315797+atouchet@users.noreply.github.com>

mrobinson commented:
Hrm. I'm not sure what happened here when I imported `devices`. The real README should be this one: https://github.com/servo/devices/blob/master/README.md. Perhaps you can replace the contents with the original from that repository? Thanks!

atouchet committed:
Replace bluetooth Readme with correct version

Signed-off-by: Alex Touchet <26315797+atouchet@users.noreply.github.com>

atouchet commented:
I have replaced the Readme using the one from the devices repo. I also made some formatting changes and replaced any references to the device crate with the bluetooth crate.

atouchet commented:
Wait, is this still accurate:

> The bluetooth-test feature is not a default feature, to use it, append features = ["bluetooth-test"], to the device crate dependency in the project's Cargo.toml.

The bluetooth crates added `default = ["bluetooth-test"]`

Should that line just be removed then?

mrobinson commented:
> Should that line just be removed then?

Yes, I think it should probably be removed.

atouchet committed:
Update Readme text

Signed-off-by: Alex Touchet <26315797+atouchet@users.noreply.github.com>

atouchet commented:
Okay I think this is ready for review.

jdm reviewed (approved):
Thanks!
